packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -219,6 +219,10 @@ abstract contract CrossDomainMessenger is
         external
         payable
     {
+        // On L1 this function will check the Portal for its paused status.
+        // On L2 this function should be a no-op.
+        _revertWhenPaused();
```

Comment:
Is this necessary? Why can't we do:

```solidity
require(paused() == false, "OptimismPortal: paused");
```

Reply:
My thinking was that calling a function the just returns is more likely to be optimized away than this.
I'm not certain either way, might do a little sandbox test, or else just take your suggestion.

Reply:
I verified this.